### PR TITLE
comp: button

### PR DESCRIPTION
**Pull request by jsuryakt**
Please review button component

image

### CODE REVIEW

**Review by DevNirah7 — APPROVED** [review]
Looks good to commit!

**Review by sashaboi — APPROVED** [review]
Good to go !

File: minimal.css [hunk 1]
```diff
@@ -128,4 +128,112 @@
...
+    padding: 0.5rem 1rem;
+    font-size: 0.8rem;
+    font-weight: 700;
+    border: none;
+    cursor: pointer;
+    transition: all .2s ease-in-out;
+}
+
+.btn-primary {
+    background-color: #0070f3;
```
Comment (TJain02) [inline minimal.css]: Try using color variables.

**Review by TJain02 — COMMENTED** [review]
Sugested the changes. Rest all looks good!

**Review by shivamSspirit — COMMENTED** [review]
good to be merge

**Pull request merged**